config/projects/datadog-agent.rb

```diff
@@ -138,37 +187,35 @@
 dependency 'python-etcd'
 dependency 'pyyaml'
 dependency 'simplejson'
-dependency 'supervisor'
 dependency 'tornado'
 dependency 'uptime'
 dependency 'uuid'
-dependency 'zlib'
 
-# Check dependencies
+# Check dependencies (docker-py is required on Windows too because we import it in
+# dd-agent/utils/platform.py indirectly, since docker - and therefore - docker-py should
+# exist on Windows at some point, we're just anticipating on things anyway :) )
 dependency 'adodbapi'
 dependency 'beautifulsoup4'
 dependency 'dnspython'
+dependency 'docker-py'
```

Comment:
`docker-py` should stay in the `Agent dependencies` since it's not used only in the checks but also in the core for all the service discovery-related stuff